apps/src/applab/applab.js

```diff
@@ -302,7 +306,10 @@ function queueOnTick() {
 
 function onDebugInputKeyDown(e) {
   if (e.keyCode == KeyCodes.ENTER) {
+    e.preventDefault();
```

Comment:
Is there a particular reason we need to call `preventDefault` here?  I'm not sure there's a default enter key event for a content-editable div.